Makefile

```diff
@@ -30,7 +30,7 @@ COVERAGE_REPORT := $(COVERAGE_DIR)/coverage.html
 GOOS := $(shell go env GOOS)
 GOARCH := $(shell go env GOARCH)
 
-GO_LD_FLAGS := -ldflags="-s -w -X github.com/tetratelabs/getenvoy/pkg/version.version=$(GETENVOY_TAG)"
+GO_LD_FLAGS := -ldflags="-s -w -X github.com/tetratelabs/getenvoy/internal/version.Current=$(GETENVOY_TAG)"
```

Comment:
@llinder FYI: this can override variables at build time, but not const (var, not const)